docs/source/guide/recommender_models.rst

```diff
@@ -0,0 +1,80 @@
+Recommender Models
+=====================
+
+This guide explains the different type of recommender models with a focus on deep-learning based ones.
+
+.. contents:: Table of Contents
+   :local:
+   :backlinks: none
+.. section-numbering::
```

Comment:
I have a personal preference against numbering sections because they can change quickly.  If you have a strong preference to add numbering, please apply to the whole docset.

I'd also remove "Table of Contents", for a cleaner, crisper look, but completely up to you.

Reply:
sounds good